packages/marketplace/contracts/exchange/ExchangeCore.sol

```diff
@@ -295,56 +270,62 @@ abstract contract ExchangeCore is Initializable, ContextUpgradeable, TransferExe
         uint256 takeBuyAmount = newFill.rightValue;
         uint256 makeBuyAmount = newFill.leftValue;
 
-        if (((_msgSender() != msg.sender) && !nativeMeta) || ((_msgSender() == msg.sender) && !nativeOrder)) {
+        // TODO: this force me to pass from, do we want it ?
```

Comment:
I think we do. @mvanmeerbeck ?

Reply:
all of that is for native eth, which we're removing so, lets just ignore